crates/sui-types/src/crypto.rs

```diff
@@ -234,26 +138,19 @@ pub fn get_key_pair_from_rng<R>(csprng: &mut R) -> (SuiAddress, KeyPair)
 where
     R: rand::CryptoRng + rand::RngCore,
 {
-    let kp = DalekKeypair::generate(csprng);
-    let keypair = KeyPair {
-        key_pair: kp,
-        public_key_cell: OnceCell::new(),
-    };
-    (SuiAddress::from(keypair.public_key_bytes()), keypair)
+    let kp = KeyPair::generate(csprng);
+    (kp.public().to_address(), kp)
 }
 
-// TODO: C-GETTER
-pub fn get_key_pair_from_bytes(bytes: &[u8]) -> (SuiAddress, KeyPair) {
-    let keypair = KeyPair {
-        key_pair: DalekKeypair::from_bytes(bytes).unwrap(),
-        public_key_cell: OnceCell::new(),
-    };
-    (SuiAddress::from(keypair.public_key_bytes()), keypair)
-}
+// // TODO: C-GETTER
```

Comment:
is this removed because narwhal supports this already?

Reply:
It is removed because Narwhal currently does not support it, I can also add this to Narwhal and uncomment this for the PR.

Reply:
I added it back in.